src/main/kotlin/no/skatteetaten/aurora/cantus/controller/NexusController.kt

```diff
@@ -0,0 +1,38 @@
+package no.skatteetaten.aurora.cantus.controller
+
+import no.skatteetaten.aurora.cantus.service.NexusClient
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+
+data class Version(
+    val name: String,
+    val lastModified: String
+)
+
+@RestController
+class NexusController(val nexusClient: NexusClient) {
+
+    @GetMapping("/versions")
+    fun getVersions(
+        @RequestParam imageGroup: String,
+        @RequestParam name: String
+    ): Flux<Version> {
```

Comment:
Her brukes ikke den felles response konvolutten som vi bruker ellers i andre fraggler. Skal vi gå bort ifra å ha en felles konvolutt? Hva tenker du @jarlehansen ? Det har jo ikke blitt prioritert å samkjøre response formatene

Reply:
Kanskje vi burde ha det som et tema på teammøtet?

Reply:
Jeg kan legge den til her hvis dere mener det er bedre slik, men jeg kan heller ikke si at jeg utviser stor entusiasme overfor ting som er mer kompliserte enn de trenger å være.

Reply:
Jeg har diskutert dette med Jarle nå. Konklusjonen er et er mye arbeid å legge til dette, og vi får ikke noe særlig igjen for det. Sånn sett er det kanskje bedre å fjerne det fra den andre controlleren. Men det er uansett noe som egner seg bedre til en separat oppgave, sånn at vi kan få inn disse endringene nå.